src/libserver/rspamd_symcache.c

```diff
@@ -3384,6 +3384,27 @@ rspamd_symcache_set_symbol_flags (struct rspamd_symcache *cache,
 	return FALSE;
 }
 
+void
+rspamd_symcache_get_symbol_details(struct rspamd_symcache *cache,
+										   const gchar *symbol,
+										   const ucl_object_t *this_sym_ucl)
+{
+	struct rspamd_symcache_item *item;
+
+	g_assert (cache != NULL);
+	g_assert (symbol != NULL);
+
+	item = rspamd_symcache_find_filter (cache, symbol, false);
+
+	if (item) {
+		ucl_object_insert_key (this_sym_ucl,
+			ucl_object_fromstring(item->type_descr),
+			"type", strlen("type"), false);
```

Comment:
You can use `0` to indicate zero terminated key "type" here. It's not a problem to call `strlen` though.

Reply:
Just to confirm, you are suggesting to replace strlen(“type”) with 0 in this call?
Will leave as is for now — please change as part of merge to make it more efficient to eliminate the strlen call.

Reply:
It will not make any difference in efficiency, as `strlen` from a literal is automatically substituted by a compiler.